redhat-developer/developers.redhat.com
# DEVELOPER-5682 - Add PFE Layouts library and Red Hat Theme CSS

### JIRA Issue Link
* https://issues.jboss.org/browse/DEVELOPER-5682

### Verification Process
* Navigate to any page
* Open Dev Tools and inspect an element
* Should see the list of redhat-theme variables and should be able to apply PFE Layout classes to content to create grid layout.
